java/swagger-petstore/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8-jre-alpine
```

Comment:
is this part of requirement? do we need a docker image? is this tested?

Reply:
It has been tested! Need to validate if it's necessary.